packages/algob/test/fixture-projects/config-project/assets/gold-asa.py

```diff
@@ -0,0 +1,31 @@
+
+from pyteal import *
+
+"""GOLD Transfer"""
+''' Accepts only if (transaction type is OPT-IN OR (transaction type is asset transfer, 
+    sender is goldOwnerAccount and asset transfer amount is less than equal to 1000 ))'''
+
+asset_amt = Int(1000)
+tmpl_sen = Addr("M7VR2MGHI35EG2NMYOF3X337636PIOFVSP2HNIFUKAG7WW6BDWDCA3E2DA")
```

Comment:
what is `sen` is not really clear. Let's rename that variable to `tmpl_sender`

Reply:
ok